src/codeu/chat/common/UserInterest.java

```diff
@@ -40,25 +40,37 @@ public UserInterest read(InputStream in) throws IOException {
         }
     };
 
-    public final Uuid id;
-    public final Uuid owner;
-    public final Uuid userId;
-    public final Time creation;
     public Set<ConversationHeader> conversations;
 
     // constructor for a User interest
```

Comment:
The contractors format is very clear, so this comment should most likely be deleted.